apps/site/src/app/(home)/sections/Landing/Characters.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+
+import waterAnteater from "@/assets/images/intro-water-anteater.svg";
+import waterAnteaterFoam from "@/assets/images/intro-character-foam.svg";
+import waterAnteaterShadow from "@/assets/images/intro-water-anteater-shadow.svg";
+import beachBall from "@/assets/images/intro-beach-ball.svg";
+
+import mainCharacter from "@/assets/images/intro-main-anteater.svg";
+
+import styles from "./Characters.module.scss";
+
+export default function Characters() {
+    return (
+        <div>
+            <Image src={waterAnteater} alt="Character" className={styles.character} />
+            <Image src={waterAnteaterShadow} alt="Character Shadow" className={styles.characterShadow} />
+            <Image src={waterAnteaterFoam} alt="Character" className={styles.characterFoam} />
```

Comment:
Thought: the alt text for these images could be more descriptive. For example, something like "anteater floating in the water."